client/state/signup/segments/actions.js

```diff
@@ -0,0 +1,29 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { SIGNUP_SEGMENTS_REQUEST, SIGNUP_SEGMENTS_SET } from 'state/action-types';
+
+import 'state/data-layer/wpcom/signup/segments';
+
+/**
+ * Action creator: Request segments data.
+ *
+ * @return {Object} The action object.
+ */
+export const requestSegments = () => ( {
```

Comment:
As `requestSegments` is not dynamic, could we just return a plain object?
```js
export const requestSegments = {
  type: SIGNUP_SEGMENTS_REQUEST,
};
```

Reply:
I don't grok what you mean here. Could you elaborate?

Right now it's a redux action creator and I plan to use the action creator in a connect function, but if we returned an object we'd still have to dispatch the action somewhere, right?

Reply:
Sorry yeah! I think I was actually wrong here!
We can dispatch plain object actions like so:
```js
const someThunk = payload => dispatch => {
  dispatch( { type: 'SOME_ACTION, payload } );
}
```
So I thought we could just do that in this case, given that the resulting action doesn't change with variables etc. but I tested this and t doesn't work when importing and binding within `connect`s `mapDispatchToProps` argument.

So ignore this comment - I was barking up a tree that turned out to be wrong :P